python/paddle/fluid/tests/unittests/test_switch_case.py

```diff
@@ -170,14 +169,13 @@ def fn_3():
                         fn_2, x=3)
                 })
             return out
-            #return layers.fill_constant(shape=[4, 3], dtype='int32', value=3)
 
         main_program = Program()
         startup_program = Program()
         with program_guard(main_program, startup_program):
-            index_1 = layers.fill_constant(shape=[1], dtype='int32', value=1)
+            index_1 = layers.data(name="index_1", shape=[1], dtype='uint8')
```

Comment:
Not important.

Unlike fluid.data, layers.data will append -1 as batch size. You feed the array with shape [1] so it didn't have error. But I would suggest to change "layers.data" to "fluid.data" to de-ambigurity

Reply:
Thanks. Done.